src/index.html

```diff
@@ -10,5 +10,159 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <span><nav>HOME</nav></span>
+    <span><button>DOWNLOAD MY FULL RESUME</button></span>
+    <div><h1>John Roberts</h1></div>
```

Comment:
Might worth to have a look at the lesson 1 PPT to find the difference between inline and block elements.

h1 is block element and it doesn't need the  wrapping div tag.